lua/nvim-tree/actions/moves/git.lua

```diff
@@ -0,0 +1,62 @@
+local explorer_node = require "nvim-tree.explorer.node"
+local lib = require "nvim-tree.lib"
+local moves_item = require "nvim-tree.actions.moves.item"
+
+local M = {}
+local MAX_DEPTH = 100
+
+function M.next_item_recursive()
```

Comment:
This could be generic - we could move this into `item.lua` to apply to all the 3 similar moves: git, diagnostics and opened.

Reply:
I agree. I'll look into making this more generic.